lms/templates/help_modal.html

```diff
@@ -76,15 +76,9 @@
 % else:
 	<p>${_('Here are some of the most popular articles from the Help Center:')}</p>
 	<ul class="help-buttons">
-	  <li><a href="https://stanfordonline.zendesk.com/hc/en-us/articles/202275024-How-do-I-create-an-account-" target="_blank">
-		${_('How to register an account')}
-	  </a></li>
-	  <li><a href="https://stanfordonline.zendesk.com/hc/en-us/articles/201864484-How-do-I-reset-my-password-" target="_blank">
-		${_('How to reset a password')}
-	  </a></li>
-	  <li><a href="https://stanfordonline.zendesk.com/hc/en-us/articles/203009244-How-do-I-enroll-in-a-course-" id="howto-enroll-link" target="_blank">
-		${_('How to enroll in a course')}
-	  </a></li>
+          % for link in settings.HELP_MODAL_LINKS:
```

Comment:
Do you mean this specific template? This file only gets loaded if ENABLE_FEEDBACK_SUBMISSION is enabled